glean_parser/schemas/pings.2-0-0.schema.yaml

```diff
@@ -175,6 +175,14 @@ additionalProperties:
       additionalProperties:
         type: string
 
+    enabled:
+      title: Whether or not this ping is enabled
+      description: |
+        When `true`, the ping will be sent as usual.
+        When `false`, the ping will not be sent, but the data will continue to
+        be collected but will not be cleared when the ping is submitted.
```

Comment:
Optional parameters need to have a description that:
1) Says they're optional
2) Says what the default value will be if it's omitted

Reply:
Yeah, I was just following the existing examples in this file where only the _required_ properties are labeled as such in the description. I like having more detail though so you won't get any arguments from me over adding it here.